docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,42 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
```

Comment:
I guess this is only true if `catalog.store=file`, which is the default, but since it's not documented, maybe we can update this when documenting `catalog.store` and other properties?

Reply:
We should document these properties now as well

Reply:
```suggestion
catalogs using these SQL commands adds and removes catalog
```